quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/cli/PicocliTest.java

```diff
@@ -325,20 +342,29 @@ public void testReaugFromProdToDevExport() {
 
     @Test
     public void testNoReaugFromProdToExport() {
-        build("build");
+        build("build", "--db=dev-file");
 
         Environment.setRebuildCheck(); // will be reset by the system properties logic
-        NonRunningPicocli nonRunningPicocli = pseudoLaunch("export", "--file=file");
+        NonRunningPicocli nonRunningPicocli = pseudoLaunch("export", "--db=dev-file", "--file=file");
         assertEquals(CommandLine.ExitCode.OK, nonRunningPicocli.exitCode);
         assertFalse(nonRunningPicocli.reaug);
     }
 
+    @Test
+    public void testDBRequiredAutoBuild() {
+        build("build", "--db=dev-file");
+
+        Environment.setRebuildCheck(); // will be reset by the system properties logic
+        NonRunningPicocli nonRunningPicocli = pseudoLaunch("export", "--file=file");
+        assertEquals(CommandLine.ExitCode.OK, nonRunningPicocli.exitCode);
+    }
```

Comment:
What's the purpose of this test case?

Reply:
It's no longer valid for the deprecation scenario.